terraform/ecs_asg/variables.tf

```diff
@@ -51,3 +51,11 @@ variable "admin_cidr_ingress" {
   default     = "0.0.0.0/0"
   description = "CIDR for SSH access to EC2 instances"
 }
+
+variable "sns_topic_arn" {
+  description = "ARN of the topic where to push notifications when an EC2 instance is set to terminating state"
+}
+
+variable "publish_to_sns_policy" {
+  description = "Policy document to publish to the sns topic"
```

Comment:
Okay, could we put that comment in the description please? I think that’s a clearer way of putting it.